extensions/crossmodel-lang/src/extension.ts

```diff
@@ -35,7 +34,7 @@ function launchLanguageClient(context: vscode.ExtensionContext): LanguageClient
 
 function createServerOptions(context: vscode.ExtensionContext): ServerOptions {
    // needs to match the configuration in tsconfig.json and webpack.config.js
-   const serverModule = context.asAbsolutePath(path.join('out', 'server-main'));
+   const serverModule = context.asAbsolutePath(path.join('out', 'main.cjs'));
```

Comment:
So you can mix .js and .cjs imports? cjs is for the CommonJS files? as opposed to .js files now being ESM?

Reply:
Yes, we use `.cjs` to denote CommonJS files and assume that all other files are ESM in the extension. However, VS Code extensions only work with CommonJS code so we transpile the two necessary extension files (`extension.ts` and `main.ts`) as CommonJS in the builder (`esbuild.mjs`).